noah/src/main/resources/crawler/promed_mail.py

```diff
@@ -0,0 +1,78 @@
+import io
+import json
+import urllib2
+import urllib
+import datetime
+import gzip
+import sets
+import re
+
+def getPages(query) :
+	headers = {}
+	headers['User-Agent'] = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
+	headers['X-Requested-With'] = 'XMLHttpRequest'
+	headers['Referer'] = "http://promedmail.org/"
+	headers['Accept'] = "application/json, text/javascript, */*"
+	headers['Accept-Encoding'] = "gzip, deflate, sdch"
+	headers['Accept-Language'] = "en-US,en;q=0.8"
+	headers['Connection'] = "keep-alive"
+	headers['Content-Type'] = "application/x-www-form-urlencoded"
+	headers['Host'] = 'promedmail.org'
+	url = "http://www.promedmail.org/ajax/runSearch.php?"
+	request = urllib2.Request(url+query, headers = headers)
+	response = urllib2.urlopen(request)
+	return response
+
+def getQuery1Page1(start_date, end_date):
+	query1 = {"kwby1":'summary', 'search':'zika', 'show_us':'1', 'feed_id':'1'}
+	query1['date1'] = start_date
+	query1['date2'] = end_date
+	return query1
+	
+def getQuery2Page2(start_date, end_date):
```

Comment:
all those `getQuery*page*` is exactly the same. should it be one function just called `getNext`?